orocos_kdl/src/chainiksolverpos_nr_jl.cpp

```diff
@@ -23,100 +23,133 @@
 
 #include "chainiksolverpos_nr_jl.hpp"
 
+#include <iostream>
```

Comment:
Why is this needed? The only `std::cout` is commented. So please remove that one.